perseo-main/src/test/java/com/telefonica/iot/perseo/GenericListenerTest.java

```diff
@@ -63,7 +65,7 @@ public void tearDown() {
      */
     @Test
     public void testUpdate() {
-        System.out.println("update");
+    	logger.info("update");
```

Comment:
Review also the files in test/ directory, to fix all the wrong indentation cases like this one.

Reply:
Fixed in 61b8a4c9c7fe92f547917fd56b451a77f39db5f5